website/react-magma-docs/src/pages/design/button-group.mdx

```diff
@@ -0,0 +1,167 @@
+---
+pageTitle: Button Group Design Guidelines
+title: Button Group
```

Comment:
Under API, we use `ButtonGroup`. Do we care that they are different?

Reply:
Fixed